tests/unit_tests/test_writer_txt.py

```diff
@@ -1,32 +1,9 @@
 """Check that writing tab-delimited files works well"""
-from pathlib import Path
 
-import pytest
 import mokapot
 import pandas as pd
 
 
-def test_sanity(psms, tmp_path):
```

Comment:
Why is this removed? Should we add an alternative test?

Reply:
same as above